assets/messages.yml

```diff
@@ -351,6 +351,20 @@ messages:
         This is known to be an issue involving detection of the Java runtime.
         Please take a look at the Moderator’s Note on that ticket to see solutions to your issue.
 
+        %quick_links%
+      fillname: []
+  duplicate-of-mc-108:
+    - project: mc
+      name: Duplicate of MC-108
+      shortcut: quasi
+      message: |-
+        *Thank you for your report!*
+        We're tracking this issue as *%s%*, so this ticket is being resolved and linked as a *duplicate*.
```

Comment:
```suggestion
        We're tracking this issue as *MC-108*, so this ticket is being resolved and linked as a *duplicate*.
```

Reply:
Ah yes, that would be pretty smart to do